requirements.txt

```diff
@@ -43,7 +43,7 @@ openpyxl
 CoolProp
 attrs
 utm
-orbit-nrel @ git+https://github.com/WISDEM/ORBIT.git@3baed36052d7503de6ea6f2db0b40945d637ea35
```

Comment:
Will this someday be a tagged release that we can pin instead of the commit number or did you purposefully pin the commit instead of a tag for a specific reason?

Reply:
This is the removed code, see the change directly to the right.

![image](https://github.com/user-attachments/assets/3cc58996-7556-4263-9160-5918be29aa25)

Reply:
I've never been more embarrassed, I blame a lack of coffee, thanks Rob!